app/controllers/hyrax/autopopulation/dashboard/work_fetchers_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Autopopulation
+    module Dashboard
+      class WorkFetchersController < ApplicationController
+        before_action :ensure_authorized!
+        include ::Hyrax::ThemedLayoutController
+        with_themed_layout 'dashboard'
+
+        def index
+          add_breadcrumb t(:'hyrax.controls.home'), ::Hyrax::Engine.routes.url_helpers.root_path
+          add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), ::Hyrax::Engine.routes.url_helpers.dashboard_path
+
+          @approved_works = []
+          @draft_works = []
+        end
+
+        # Uses Hyrax::Autopopulation::RecordPersistence class via the config object
+        def settings
+          if config_object.storage_type == "activerecord"
+            config_object.persistence_class.constantize.new.save(autopopulation_settings_params, current_account)
+          else
+            config_object.persistence_class.constantize.new.save(autopopulation_settings_params)
```

Comment:
Could you make this a little more consise?

```ruby
          persistence_class = config_object.persistence_class.constantize.new
          if config_object.storage_type == "activerecord"
            persistence_class.save(autopopulation_settings_params, current_account)
          else
           persistence_class.save(autopopulation_settings_params)
          end
```

Or, even better (i think this would work): 

```ruby
    args = if config_object.storage_type == "activerecord"
             [autopopulation_settings_params, current_account]
           else
             [autopopulation_settings_params]
           end

    config_object.persistence_class.constantize.new.save(**args)
```

Reply:
ok noted, thanks

Reply:
@prdanelli I think this is a good place to talk about replacing the if condition with a hash:

````
   persistence_class = config_object.persistence_class.constantize.new

  args = {
               "activerecord" =>  [autopopulation_settings_params, current_account],
               "redis" => [autopopulation_settings_params]
             }.freeze.dig(config_object.storage_type)

    config_object.persistence_class.constantize.new.save(**args)
````

Reply:
Scratch it, the current_account will always throw an error in Hyrax app,  so I will keep the **if condition**, but atleast now you see in code what I spoke about in one of our previous Stand-ups about replacing **if** with **hash**

Reply:
This

      persistence_class.constantize.new.save(**args)

Should be one * when calling a method that expects multiple arguments but passing in an array

       persistence_class.constantize.new.save(*args)

Reply:
Thanks @mankind I always get the `**` and `*` confused when used to splat out the arguments.